MPU/src/driverio.cpp

```diff
@@ -137,4 +159,25 @@ void DriverIO::wheelIO_cb(const CAN_message_t &msg)
 
     decrButton.setButtonState(wheelio.io.button2);
     incrButton.setButtonState(wheelio.io.button4);
-}
\ No newline at end of file
+    regenButton.setButtonState(wheelio.io.button6);
+    torqueIncreasePaddle.setButtonState(wheelio.io.paddle_r);
+    torqueDecreasePaddle.setButtonState(wheelio.io.paddle_l);
+    accumulatorFanDial.setDialValue(wheelio.io.pot_l);
+    motorFanDial.setDialValue(wheelio.io.pot_r);
+}
+
+void DriverIO::handleDialState()
+{
+    float motorFanPercentage = ((float)motorFanDial.getDialValue() / 4096.0);
+    if (motorFanPercentage < 0.1) motorFanPercentage = 0.0;
+    else if (motorFanPercentage > 0.9) motorFanPercentage = 1.0;
+    gpio->setRadiatorFanPercentage(motorFanPercentage);
```

Comment:
ngl I feel a bit weird calling a GPIO function in here, but I can't think of a better way to do it. GPIO is intended to be some of the "pseudo-controller" classes that handle the logic behind it and intermingling them feels a bit odd, but again I can't really think of a better way to do it lol